dev-libs/libmspack/libmspack-0.5~alpha.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A portable library for Microsoft compression formats"
+DESCRIPTION="The following file formats are supported: \
+SZDD files, which use LZSS compression \
+KWAJ files, which use LZSS, LZSS+Huffman or deflate compression \
+.HLP (MS Help) files, which use LZSS compression \
+.CAB (MS Cabinet) files, which use deflate, LZX or Quantum compression \
+.CHM (HTML Help) files, which use LZX compression \
+.LIT (MS EBook) files, which use LZX compression and DES encryption \
+.LZX (Exchange Offline Addressbook) files, which use LZX compression \
+To determine the capabilities of the library, and the binary compatibility \
+version of any particular compressor or decompressor, use the \
+mspack_version() function. The UNIX library interface version is \
+defined as the highest-versioned library component."
+HOMEPAGE="https://www.cabextract.org.uk/libmspack/"
+COPYRIGHT="2000-2015 Stuart Caie"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/libmspack-0.5alpha.tar.gz"
+CHECKSUM_SHA256="8967f275525f5067b364cee43b73e44d0433668c39f9376dfff19f653d1c8110"
+SOURCE_DIR="libmspack-0.5alpha"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libmspack$secondaryArchSuffix = $portVersion
+	lib:libmspack$secondaryArchSuffix = $portVersion = 0.1.0 compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libmspack${secondaryArchSuffix}_devel = $portVersion
+	devel:libmspack$secondaryArchSuffix = $portVersion = 0.1.0 compat >= 0
+	"
+REQUIRES_devel="
+	libmspack$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	rm $libDir/libmspack.la
```

Comment:
Please insert this block
```

	if [ -z "$secondaryArchSuffix" ]; then
		install -d $docDir $developDocDir
		install -t $docDir AUTHORS README TODO
		cp -r doc/* $developDocDir 
		rm $developDocDir/Doxyfile*
		rm $developDocDir/Makefile*
	fi

```
between the "make install" and the "rm $libDir/libmspack.la", keeping the blank lines before and after the block, to have a nice looking recipe ;-)

It will add some docs to the base and _devel packages in case the recipe is being built for any primary arch.